tests/contrib/mako/test_mako.py

```diff
@@ -0,0 +1,71 @@
+import os.path
+import unittest
+
+# 3rd party
+from nose.tools import eq_
+from mako.template import Template
+from mako.lookup import TemplateLookup
+from mako.runtime import Context
+
+from ddtrace import Pin
+from ddtrace.contrib.mako import patch, unpatch
+from ddtrace.compat import StringIO, to_unicode
+from tests.test_tracer import get_dummy_tracer
+
+TEST_DIR = os.path.dirname(os.path.realpath(__file__))
+TMPL_DIR = os.path.join(TEST_DIR, "templates")
+
+
+class MakoTest(unittest.TestCase):
+    def setUp(self):
+        patch()
+        self.tracer = get_dummy_tracer()
+        Pin.override(Template, tracer=self.tracer)
+
+    def tearDown(self):
+        unpatch()
+
+    def test_render(self):
+        # render
+        t = Template("Hello ${name}!")
+        eq_(t.render(name="mako"), "Hello mako!")
+
+        spans = self.tracer.writer.pop()
+        eq_(len(spans), 1)
+
+        for span in spans:
```

Comment:
We should only have 1 span here, can we update to be `self.assertEqual(spans[0].service, 'mako')`, etc?